src/software/ai/hl/stp/action/autochip_move_action.cpp

```diff
@@ -0,0 +1,41 @@
+#include "software/ai/hl/stp/action/autochip_move_action.h"
+
+AutochipMoveAction::AutochipMoveAction(bool loop_forever, double close_to_dest_threshold,
+                                       Angle close_to_orientation_threshold)
+    : MoveAction(loop_forever, close_to_dest_threshold, close_to_orientation_threshold),
+      chip_distance_meters_(0.0)
```

Comment:
just curious, why do we set this to 0 here? should it not be zero when we construct the `AutochipMoveAction`

Reply:
Explicit setting of this variable vs relying on implicit setting to 0?